src/agenteval/models.py

```diff
@@ -46,7 +43,7 @@ class SubmissionMetadata(BaseModel):
 class TaskResults(BaseModel):
     """Scores for all tasks in the suite"""
 
-    results: list[TaskResult] | None = None
+    results: list[TaskResult]
```

Comment:
Given that this is the only member, it seemed unnecessary to allow it to be unspecified